FWCore/Framework/src/EventProcessor.cc

```diff
@@ -1259,11 +1259,11 @@ namespace edm {
     unsigned int streamIndex = 0;
     for (; streamIndex < preallocations_.numberOfStreams() - 1; ++streamIndex) {
       tbb::task::enqueue(*edm::make_functor_task(tbb::task::allocate_root(), [this, streamIndex, h = iHolder]() {
-        handleNextEventForStreamAsync(std::move(h), streamIndex);
+        handleNextEventForStreamAsync(h, streamIndex);
```

Comment:
This one I don’t understand

Reply:
Not sure either.
Maybe because `handleNextEventForStreamAsync(...)` takes its first argument by value, so there is no point in moving ?
https://github.com/cms-sw/cmssw/blob/4bbb63b0f11ccf880dc738a7338165db26e42503/FWCore/Framework/interface/EventProcessor.h#L270

Reply:
(just shooting out dark) Could it be that the check sees that `WaitingTaskHolder` contains only a pointer, and ignores the code in destructor and and copy/move constructors for its decision?

Reply:
That would be 'real bad' if so. Hopefully not.